tests/provisioner-tests/test/testkit/config.go

```diff
@@ -41,6 +43,12 @@ type GCPConfig struct {
 	ProjectName string `envconfig:"default=''"`
 }
 
+type DirectorClientConfig struct {
+	URL                        string `envconfig:"default=http://compass-director.compass-system.svc.cluster.local:3000/graphql"`
+	Namespace                  string `envconfig:"default=compass-system"`
+	OauthCredentialsSecretName string `envconfig:"default=compass-provisioner-credentials"`
+}
+
 func (c TestConfig) String() string {
```

Comment:
Please add new params here, so that they get printed.

Reply:
Added!